dataprofiler/tests/profilers/profiler_options/test_unstructured_text_profile_options.py

```diff
@@ -0,0 +1,138 @@
+        expected_error = ("type object 'is_case_sensitive' has no attribute "
+                          "'is_enabled'")
+        with self.assertRaisesRegex(AttributeError, expected_error):
+            option.set({'is_case_sensitive.is_enabled': True})
+
+        expected_error = ("type object 'stop_words' has no attribute "
+                          "'is_enabled'")
+        with self.assertRaisesRegex(AttributeError, expected_error):
+            option.set({'stop_words.is_enabled': True})
+
+    def test_validate_helper(self):
+        super(TestTextProfilerOptions, self).test_validate_helper()
+
+    def test_validate(self):
+
+        super(TestTextProfilerOptions, self).test_validate()
+
+        params_to_check = [
+            # non errors
+            dict(prop='is_enabled', value_list=[False, True], errors=[]),
+            dict(prop='is_case_sensitive', value_list=[False, True], errors=[]),
+            dict(prop='stop_words',
+                 value_list=[None, ['word1', 'word2'], []],
+                 errors=[]),
+            dict(prop='words', value_list=[False, True], errors=[]),
+            dict(prop='vocab', value_list=[False, True], errors=[]),
+
+            # errors
+            dict(prop='stop_words',
+                 value_list=[2, 'a', [1, 2]],
+                 errors=[
+                     "TextProfilerOptions.stop_words must be None "
+                     "or list of strings."
+                 ]),
+        ]
+
+        # # this code can be abstracted to limit code everywhere else
+        # # AKA, for loop below could be abstracted to a utils func
+
+        # Default configuration is valid
+        option = self.get_options()
+        self.assertIsNone(option.validate(raise_error=False))
+
+        for params in params_to_check:
+            prop, value_list, expected_errors = (
+                params['prop'], params['value_list'], params['errors']
+            )
+            option = self.get_options()
+            for value in value_list:
+                if prop not in ['words', 'vocab']:
+                    setattr(option, prop, value)
+                else:
+                    prop_enable = '{}.is_enabled'.format(prop)
+                    setattr(option, prop_enable, value)
+
+                validate_errors = option.validate(raise_error=False)
+                if expected_errors:
+                    self.assertListEqual(
+                        expected_errors,
+                        validate_errors,
+                        msg='Errored for prop: {}, value: {}.'.format(prop,
+                                                                      value))
+                else:
+                    self.assertIsNone(
+                        validate_errors,
+                        msg='Errored for prop: {}, value: {}.'.format(prop,
+                                                                      value))
+
+        # this time testing raising an error
+        option.stop_words = 'fake word'
```

Comment:
this only succeeds because the last param_to_check is `stop_words`
I suggest adding: `option = self.get_options()` prior to this change which will prevent other issues from occurring.

Reply:
added